docs/developer-center/guides/08-Helper-Methods-Part-2.md

```diff
@@ -153,18 +153,18 @@ Map(
 
 #### Example 1: Population Density in Dallas County
 
-In the example below, we will modify the visualization parameters number of `bins` and `pallete` to customize a choropleth map of population density in Dallas County, Texas. 
+In the example below, we will modify the visualization parameters number of `bins` and `pallete` to customize a choropleth map of population density in Dallas County, Texas.
 
 Customizations:
-- set the number of `bins` to `7` 
+- set the number of `bins` to `7`
 - custom sequential color palette (note that the three colors provided were interpolated to provide 7 colors for each class break)
 
 ```py
-from cartoframes.auth import set_default_context
+from cartoframes.auth import set_default_credentials
 from cartoframes.viz import Map
 from cartoframes.viz.helpers import color_bins_layer
 
-set_default_context('https://cartovl.carto.com/')
+set_default_credentials('https://cartovl.carto.com/')
```

Comment:
All the `set_default_credentials('https://cartovl.carto.com/')` should be changed to `set_default_credentials('carto-vl')`